python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -3,106 +3,274 @@
+                ])
+    # AIX specific deps.
+    elif platform_system == 'aix':
+        # This is the standard list of deps for AIX 5.3.
+        expected_deps = [
+            '/unix',
+            'libbsd.a',
+            'libc.a',
+            'libcrypt.a',
+            'libcrypto.a',
+            'libdl.a',
+            'libnsl.a',
+            'libpthreads.a',
+            'libpthreads_compat.a',
+            'libssl.a',
+            'libtli.a',
+            'libz.a',
+            ]
+        # sys.platform could be 'aix5', 'aix6' etc.
+        aix_version = int(sys.platform[-1])
+        if aix_version >= 7:
+            expected_deps.extend([
+                'libthread.a',
+            ])
+    # Solaris specific deps.
+    elif platform_system == 'sunos':
+        # This is the standard list of deps for a Solaris 10 build.
+        expected_deps = [
+            'libaio.so.1',
+            'libc.so.1',
+            'libcrypt_i.so.1',
+            'libcrypto.so.0.9.7',
+            'libcrypto_extra.so.0.9.7',
+            'libdl.so.1',
+            'libdoor.so.1',
+            'libgen.so.1',
+            'libintl.so.1',
+            'libm.so.2',
+            'libmd.so.1',
+            'libmp.so.2',
+            'libnsl.so.1',
+            'librt.so.1',
+            'libscf.so.1',
+            'libsocket.so.1',
+            'libsqlite3.so.0',
+            'libssl.so.0.9.7',
+            'libssl_extra.so.0.9.7',
+            'libuutil.so.1',
+            'libz.so.1',
+            ]
+    # OS X specific deps.
+    elif platform_system == 'darwin':
+        # This is the minimum list of deps for OS X.
+        expected_deps = [
+            'ApplicationServices.framework/Versions/A/ApplicationServices',
+            'Carbon.framework/Versions/A/Carbon',
+            'CoreFoundation.framework/Versions/A/CoreFoundation',
+            'CoreServices.framework/Versions/A/CoreServices',
+            'SystemConfiguration.framework/Versions/A/SystemConfiguration',
+            'libSystem.B.dylib',
+            'libcrypto.0.9.8.dylib',
+            'libgcc_s.1.dylib',
+            'libssl.0.9.8.dylib',
+            'libz.1.dylib',
+            ]
+    return expected_deps
+
+
+def get_actual_deps():
+    """
+    Here we get the list of actual deps using a shell script helper.
+    """
+    global exit_code
+    try:
+        actual_deps = subprocess.check_output('./get_binaries_deps.sh',
+                      shell=True).split()
+    except:
+        print 'Could not determine the deps for the new binaries.'
+        exit_code = 13
+    else:
+        return actual_deps
+
+
+def check_deps(expected_deps, actual_deps):
+    """
+    We check actual deps one by one to see if there is a substring of each
+    of them in any dep from the list of expected deps. This is so that an
+    actual dep of libssl.so.0.9.8 or libssl.so.1.0.0 matches an expected dep of
+    libssl.so when checking for OpenSSL.
+    """
+    global exit_code
```

Comment:
global are bad...

you can return True or False if deps are valid or not.

You can rename `check_deps` into `are_deps_valid` to make it clear that it will return a boolean value and should be used in a test.